cognite_toolkit/cognite_modules/examples/example_pump_asset_hierarchy/transformations/pump_asset_hierarchy-load-collections_pump.sql

```diff
@@ -56,4 +56,4 @@ SELECT
   LifeCycleStatus,
   LocationDescription
   ) as metadata
-from pump_assets.`collections_pump`
+from `{{raw_db}`.`collections_pump`
```

Comment:
I think this is missing from the `default.config.yam` based on the change in the snapshot.

Reply:
There was a typo...